src/tools/analyzer.md

```diff
@@ -9,24 +9,45 @@ The static analyzer evaluates your Dart code,
 checking for errors and warnings that are specified in the
 [Dart Language Specification](https://www.dartlang.org/docs/spec/).
 
-Many tools in the Dart ecosystem use the
-[analyzer package](https://pub.dartlang.org/packages/analyzer) to
-perform static analysis:
+Many tools in the Dart ecosystem use the [analysis
+server](https://github.com/dart-lang/sdk/tree/master/pkg/analysis_server)
+to perform static analysis. For example:
 
-* Some IDEs (such as [WebStorm](https://webdev.dartlang.org/tools/webstorm),
-  [IntelliJ](https://www.dartlang.org/tools/jetbrains-plugin),
+* Dart and Flutter plugins for IDEs (such as
+  [WebStorm](https://webdev.dartlang.org/tools/webstorm),
+  [IntelliJ](https://www.dartlang.org/tools/jetbrains-plugin), and
   [Atom](https://atom.io/packages/dartlang))
-* [Dart Dev Compiler (DDC)](https://github.com/dart-lang/sdk/tree/master/pkg/dev_compiler)&mdash;Experimental development tool and transpiler for web apps
 * [DartPad](https://www.dartlang.org/tools/dartpad)&mdash;browser-based
   Dart playground
+* [Dart Dev Compiler (DDC)](https://github.com/dart-lang/sdk/tree/master/pkg/dev_compiler)&mdash;experimental development tool and transpiler for web apps
 * [`dartanalyzer`](https://github.com/dart-lang/sdk/tree/master/pkg/analyzer_cli#dartanalyzer
-)&mdash;command-line tool for static analysis for web or VM apps
-* [`flutter analyze`](https://flutter.io/debugging/#the-dart-analyzer)&mdash;static analysis for mobile (Flutter) apps
+)&mdash;command-line static analyzer for web or VM apps
+* [`flutter analyze`](https://flutter.io/debugging/#the-dart-analyzer)&mdash;command-line
+  static analyzer for mobile (Flutter) apps
 
-## Customize static analysis
+## Customizing static analysis
 
 You can customize the behavior of the analyzer or the linter
 by adding an analysis options file (`analysis_options.yaml`) to
-the package root of your project. For further information, see
-[Customize Static Analysis](/guides/language-analysis-options).
+the package root of your project. The linter, an analyzer plugin,
+allows you to define (or exclude) rules specific to your project.
+For example, you can add the `close_sinks` rule to ensure that all
+sink methods include a `close()` call. For further information, see
+[Customize Static Analysis](/guides/language/analysis-options).
 
+## Building tools that use static analysis
+
+To add static analysis to your Dart tool, you have two
```

Comment:
Clearly I know nothing about the analysis server, btw. :) I was making stuff up. But basically we could be a lot clearer about the analysis server—where it runs, how you use it, ...